### PR TITLE
Adding README.

### PR DESCRIPTION
**Pull request by zackurben**
@jmdarling @rahatarmanahmed  LMK if I need to change anything.

### CODE REVIEW

**Comment by rahatarmanahmed** [comment]
I used Pure instead of Bootstrap

**Comment by zackurben** [comment]
@rahatarmanahmed my bad lol... failed on some good 'ol copy pasta... I'll get that changed.

**Comment by zackurben** [comment]
@rahatarmanahmed @jmdarling Done, LMK if you notice anything else, otherwise go ahead and merge this.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# VacationRoulette
+Catch a plane without the pain!
+
+## About
+VacationRoulette is a service that allows for one to find a vacation quickly without much planning. We decided to use the Sabre API to plan a spur of the moment, round-trip, vacation. With the information gathered from [Sabre](https://developer.sabre.com/page), we created dynamic [Kayak](http://www.kayak.com/) URLs to book the trip using the provider of your choice, for the lowest cost. VacationRoulette requires only a leaving/returning date and an Airport code, and provides minimal filters such as: general price point, desired theme, and relative location. Once a suggestion has been made, you may book a trip using that information, or continue looking for new trips.
+
+This is an opensource project under the MIT license, and utilizes an Expressjs environment on top Nodejs:
```
Comment (jmdarling) [inline README.md]: "on top of Nodejs"

**Pull request merged**